backend/src/main/scala/cromwell/backend/io/SharedFileSystem.scala

```diff
@@ -159,26 +154,25 @@ trait SharedFileSystem extends PathFactory { this: BackendJobExecutionActor =>
    *
    * @param toDestPath function specifying how to generate the destination path from the source path
    * @param strategies strategies to use for localization
-   * @param postProcessor optional function to be applied to the path after the file it points to has been localized (defaults to noop)
    * @param wdlValue WdlValue to localize
    * @return localized wdlValue
    */
-  def localizeWdlValue(toDestPath: (String => PathsPair), strategies: Stream[LocalizationStrategy], postProcessor: Option[Path => Path] = None)(wdlValue: WdlValue): Try[WdlValue] = {
+  private def localizeWdlValue(toDestPath: (String => PathsPair), strategies: Stream[LocalizationStrategy])(wdlValue: WdlValue): Try[WdlValue] = {
 
     def localize(source: Path, dest: Path) = strategies map { _(source, dest) } find { _.isSuccess } getOrElse {
       Failure(new UnsupportedOperationException(s"Could not localize $source -> $dest"))
     }
 
     def adjustArray(t: WdlArrayType, inputArray: Seq[WdlValue]): Try[WdlArray] = {
-      val tryAdjust = inputArray map localizeWdlValue(toDestPath, strategies, postProcessor)
+      val tryAdjust = inputArray map localizeWdlValue(toDestPath, strategies)
 
       TryUtil.sequence(tryAdjust, s"Failed to localize files in input Array ${wdlValue.valueString}") map { adjusted =>
         new WdlArray(t, adjusted)
       }
     }
 
     def adjustMap(t: WdlMapType, inputMap: Map[WdlValue, WdlValue]): Try[WdlMap] = {
-      val tryAdjust = inputMap mapValues { localizeWdlValue(toDestPath, strategies, postProcessor) }
+      val tryAdjust = inputMap mapValues { localizeWdlValue(toDestPath, strategies) }
```

Comment:
Fair point, although we use it so many times throughout Cromwell that if it really was a problem we very likely would have seen it already IMO